src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -402,6 +421,67 @@ export default class ArrayField extends React.Component {
             const useCardStyling = notLastOrMultipleRows;
             const CardOrDiv = useVaCards && useCardStyling ? VaCard : 'div';
 
+            const displayFields = confirmRemoveDisplayFields
+              .map(dataPath => {
+                const keys = dataPath.split('.');
+                let value = formData?.[index];
+                keys.forEach(key => {
+                  value = value ? value[key] : undefined;
+                });
+
+                if (dataPath === 'serviceBranch') {
+                  value = serviceLabels[value] || value; // Use the label if available
+                }
+
+                return { dataPath, value };
+              })
+              .filter(({ value }) => value !== undefined);
+
+            const formattedSentenceParts = displayFields
+              .filter(
+                ({ dataPath }) =>
+                  dataPath !== 'dateRange.from' && dataPath !== 'dateRange.to',
+              )
+              .map(({ dataPath, value }) => (
+                <strong key={dataPath}>{` ${value}`}</strong>
+              ));
+
+            const formattedSentence = formattedSentenceParts.reduce(
+              (acc, part, sentenceIndex) => {
+                if (sentenceIndex === formattedSentenceParts.length - 1) {
+                  return acc.concat(part);
+                }
+                return acc.concat(part, ', ');
+              },
+              [],
+            );
+
+            const dateRangeFrom = displayFields.find(
```

Comment:
can we instead just let the consumer define the whole sentence? and give them a function/callback with anything they need in order to format it instead? for example provide formData and/or anything else, then just append whatever the consumer provides. we want to try to avoid any implementation details here, or knowledge of what the consumer wants to do.